atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -259,7 +260,11 @@ public void compactInternally() {
     }
 
     private String getPrimaryKeyConstraintName(String tableName) {
```

Comment:
We have a class called prefixed table names, oracle table name getter. Another one for constraint names will seem off. Will refactor to consolidate all in a separate PR.